src/main/java/com/mraof/minestuck/client/util/MSKeyHandler.java

```diff
@@ -91,15 +92,27 @@ public static void onKeyInput(InputEvent event)	//This is only called during the
 	@SubscribeEvent
 	public static void onTick(TickEvent.ClientTickEvent event)
 	{
+
 		if(InputMappings.isKeyDown(Minecraft.getInstance().getMainWindow().getHandle(), captchaKey.getKey().getKeyCode()) && !captchaKeyPressed) {
 
 			Screen screen = Minecraft.getInstance().currentScreen;
-			if(screen instanceof ContainerScreen<?> && screen.getFocused() == null && !(screen instanceof CreativeScreen))
+
+			if(screen instanceof ContainerScreen<?> && ((ContainerScreen<?>) screen).getSlotUnderMouse() != null && !(screen instanceof CreativeScreen))
```

Comment:
The slot under mouse is already checked below. It doesn't have to happen here as well.